go.mod

```diff
@@ -15,3 +15,5 @@ require (
 	gopkg.in/go-playground/validator.v9 v9.25.0
 	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce
 )
+
+go 1.13
```

Comment:
was this change intentional/needed?

Reply:
I found that this line is automatically added whenever I run the test.
I have removed it.